### PR TITLE
Update the disconnect on linux. the release_desktop is not needed.

### PR DESCRIPTION
**Pull request by maxcapodi78**
Updated the max chunk size to 256MB

### CODE REVIEW

File: pyaedt/rpc/rpyc_services.py [hunk 1]
```diff
@@ -42,10 +42,11 @@ def on_disconnect(self, connection):
...
         if self.app:
-            try:
-                self.app[0].release_desktop()
-            except:
-                pass
+            if not os.name == "posix":
+                try:
+                    self.app[0].release_desktop()
+                except:
+                    pass
```
Comment (MaxJPRey) [inline pyaedt/rpc/rpyc_services.py:49]: Maybe we can raise an exception here to say that the desktop was not released properly.
`raise Exception("The desktop was not released properly.") `

File: pyaedt/common_rpc.py [hunk 2]
```diff
@@ -11,7 +11,9 @@
 import rpyc
 from rpyc.utils.server import ThreadedServer
 from pyaedt.rpc.rpyc_services import GlobalService
+import rpyc.core.consts
 
+rpyc.core.consts.STREAM_CHUNK = 256000000
```
Comment (MaxJPRey) [inline pyaedt/common_rpc.py:16]: Maybe just add a small comment to explain why we must set STREAM_CHUNK to 256000000.

Comment (maxcapodi78) [reply pyaedt/common_rpc.py:17]: Done. Thanks

File: pyaedt/rpc/rpyc_services.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/rpc/rpyc_services.py]: I've removed the try except since should be handled by the release destkop

**Pull request merged**